assets/stylesheets/index.css

```diff
@@ -0,0 +1,248 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #eee;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  width: 100%;
+  height: auto;
+  margin-top: 0.6em;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.signin-btn {
+  width: 7.8em;
+  height: 35px;
+  border-radius: 5px;
+  padding-right: 8px;
+  margin-left: 10px;
+}
+
+.signup img {
+  display: inline-block;
+  width: 16px;
+  height: 12px;
+  margin-top: 2px;
+  margin-right: 4px;
+  vertical-align: text-top;
+}
+
+.mini-header {
+  margin-top: 20px;
+}
+
+.header img {
+  display: inline-block;
+  width: 16px;
+  height: 12px;
+  margin-top: 2px;
+  margin-right: 4px;
+  vertical-align: text-top;
+}
+
```

Comment:
[Required] We have to avoid the repetition of code to make our project professional. Follow the DRY principle. [Pathright requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png). Reference this [article](https://speckyboy.com/good-bad-css-practices)